src/apps/applib/lib/kml.c

```diff
@@ -107,56 +107,56 @@ void kmlBoundaryFooter(void) {
     printf("</kml>\n");
 }
 
-void outputLatLonKML(const GeoPoint* g) {
-    printf("            %8lf,%8lf,5.0\n", H3_EXPORT(radsToDegs)(g->lon),
+void outputLatLngKML(const LatLng* g) {
```

Comment:
Nit: This one is mildly confusing, since KML follows `lng, lat` order, and that's what this function outputs. I'd go `Coordinate` here, as that's the KML entity being printed.

Reply:
fixed!